.github/workflows/CI.yaml

```diff
@@ -22,8 +22,6 @@ jobs:
             ~/.gradle/caches
             ~/.gradle/wrapper
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: |
```

Comment:
I guess this is the force cache refresh in CI, right?

Reply:
Yes. The problem I found (that we have in other branches as well) is that we'll have a revision for the .gradle files (which means dependencies, gradle version, etc). For example for this branch since I touched .gradle files, the hash could be `xyz`, but if the cache does not find that key linux-gradle-xyz, this line was making it default to the latest hash available for `linux-gradle`, which would still have old dependencies